mitodl/open-discussions
# Add security config and entityID setting

#### What are the relevant tickets?
- #763 

#### What's this PR do?
- Makes the SAML EntityID configurable via env settings
- Adds an option to set the SAML `wantAssertionsEncrypted` attribute to True or False

#### How should this be manually tested?
- Set the following in your `.env` file:
  - `SOCIAL_AUTH_SAML_SECURITY_ENCRYPTED=True` 
  - `SOCIAL_AUTH_SAML_SP_ENTITY_ID=http://od.odl.local:8063/saml/metadata`
- Go to `/saml/metdata`
  - The XML should include `entityID="http://od.odl.local:8063/saml/metadata"` on the 1st line, as well as a `<md:KeyDescriptor use="encryption">` element.
